build.gradle

```diff
@@ -21,6 +21,7 @@ java {
 
 repositories {
   mavenCentral()
+  mavenLocal()
```

Comment:
Did you intend to check in this addition?

Reply:
Yes, as I wasn't on the VPN and thus couldn't resolve 7.0-SNAPSHOT from Artifactory. Don't want the VPN to be a requirement for doing stuff.